ginisdk/src/main/java/net/gini/android/SdkBuilder.java

```diff
@@ -181,23 +219,52 @@ public Gini build() {
      * Helper method to create (and store) the RequestQueue which is used for both the requests to the Gini API and the
      * Gini User Center API.
      *
-     * @return                      The RequestQueue instance.
+     * @return The RequestQueue instance.
      */
     private synchronized RequestQueue getRequestQueue() {
         if (mRequestQueue == null) {
-            RequestQueueBuilder requestQueueBuilder = new RequestQueueBuilder(mContext);
             if (mCache != null) {
-                requestQueueBuilder.setCache(mCache);
+                mRequestQueue = new RequestQueue(mCache, new BasicNetwork(getHttpStack()));
+            } else {
+                mRequestQueue = Volley.newRequestQueue(mContext, getHttpStack());
             }
-            mRequestQueue = requestQueueBuilder.build();
         }
         return mRequestQueue;
     }
 
+    /**
+     * Helper method to create a HttpStack which is used performs Http request. Optionally it can
+     * take care of certificate pinning functionality.
+     *
+     * @return HttpStack instance.
+     */
+
+    private synchronized HttpStack getHttpStack() {
+        HttpStack stack;
```

Comment:
That way you initializes the object twice, but maybe in this case it is worth to have it more clear since it is not a big deal the `HttpStack` initialization. 👍